odoo/addons/base/upgrades/1.3/pre-migrate.py

```diff
@@ -0,0 +1,5 @@
+from odoo.upgrade import util
+
+
+def migrate(cr, version):
+    util.modules.remove_module(cr, "web_tour_recorder")
```

Comment:
This script will not be executed since base is already in version `1.3`, try moving it to the upgrade repo